docs/source/AdministratorGuide/Tutorials/basicTutoSetup.rst

```diff
@@ -0,0 +1,731 @@
+.. _tuto_basic_setup:
+
+====================
+Basic Tutorial setup
+====================
+
+Tutorial goal
+=============
+
+The aim of the tutorial is to have a self contained DIRAC setup. You will be guided through the whole installation process both of the server part and the client part.
+By the end of the tutorial, you will have:
+
+* a Configuration service, to serve other servers and clients
+* a ComponentMonitoring service to keep track of other services and agents installed
+* a SystemAdministrator service to manage the DIRAC installation in the future
+* the WebApp, to allow for web interface access
+
+The setup you will have at the end is the base for all the other tutorials.
+
```

Comment:
Please just clarify briefly the connection with https://dirac.readthedocs.io/en/latest/AdministratorGuide/InstallingDIRACService/index.html

Reply:
Good idea, I will add it in a section `More links` like for the other tuto